### DISCUSSION TITLE
General Discussion: UMD-SBI Fire Growth Case

### DISCUSSION THREAD

**Discussion by dushyant-fire** [MaCFP/macfp-db#424] [General]
Hello everyone,
First, thanks to everyone who submitted their contributions to the UMD-SBI fire-growth case. I am one of the session chairs for this case and creating this thread to reach out about some results that could not be submitted via GitHub and to facilitate discussions. This will help us put together a summary presentation.

There are six submissions so far and will tag at least one contributor from each submission to this discussion. Please let me know if I should include someone else. Currently, there is one request from all who contributed, 

- @soroush-95, @TristanHehnen ,  @guoxiangzhao , @drjfloyd , and @gmaragko82 - Can you please send a representative image for your domain setup and flame structure (defined as an iso-surface of 200 kW/m3 HRRPUV)? Preferably email these separately (snapshots of instantaneous flame shapes at 30 s, 105 s, 145 s, and 185 s), with file names following the same convention as the submissions,  at dushyant.chaudhari@ul.org and alexander.snegirev@ugent.be. I think posting them here will add images to the repo which might not be desired, but I am adding one image here for an example flame shape at 30 s.

![FSRI-UMD_ISOF_30s_1cm](https://github.com/MaCFP/macfp-db/assets/17277276/e5f08571-22c1-4cfe-a02e-e16ea61bf9f6)

Happy to answer any questions. 

Thank you for your effort!
Dushyant

**Comment by drjfloyd** [comment]
Do you want one set of images for all grid resolutions and other sensitivities performed?

**Comment by dushyant-fire** [comment]
Good question. For now, please only submit set of images for one simulation case that, as per your judgement, has closest agreement with the experimental HRR. If we need additional images we will reach out.
